### PR TITLE
Feature/add supply als page2

### PR DESCRIPTION
**Pull request by Anmavel**

### CODE REVIEW

File: backend/src/main/java/com/example/backend/model/PetDTO.java [hunk 1]
```diff
@@ -5,6 +5,6 @@
 public record PetDTO(String name,
                      String nameOfBreed,
                      String photo,
-                     List<String> supplies) {
+                     List<Supply> supplies) {
```
Comment (timlenny) [inline backend/src/main/java/com/example/backend/model/PetDTO.java:9]: Remove empty space

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -19,7 +21,8 @@ function App() {
                 <Route path={"/pets"} element={<PetGallery pets={pets} navigateTo={"/pets/add"}/>}/>
                 <Route path={"/pets/add"} element={<AddPet  onAdd={postNewPet}/>}/>
                 <Route path={"/pets/:id"} element={<PetDetails pets={pets} deletePet={deletePet}/>}/>
-                <Route path={"/pets/:id/update"} element={<UpdatePet onUpdate={updatePet} pets={pets}/>} />
+                <Route path={"/pets/:id/update"} element={<UpdatePet onUpdate={updatePet} pets={pets}/>}/>
```
Comment (timlenny) [inline frontend/src/App.tsx:24]: Maybe use the /:id at the end of the path (/pets/update/:id)

**Review by timlenny — APPROVED** [review]
Verry good code! 👍 Keep it up 🎉

File: frontend/src/page/PetDetails.css [hunk 3]
```diff
@@ -24,6 +24,22 @@
...
 }
 
-.pet-details button:hover {
+.button:hover {
     background-color: darkred;
-}
\ No newline at end of file
+}
+
+.button2 {
```
[the remark this answers is not in the document]
Comment (Anmavel) [reply frontend/src/page/PetDetails.css:31]: that´s true :-)

**Pull request merged**